PED/N_0x76bba2cee66d47e9.md

```diff
@@ -1,15 +1,17 @@
 ---
 ns: PED
+apiset: client
 ---
 ## _0x76BBA2CEE66D47E9
 
 ```c
 // 0x76BBA2CEE66D47E9
-Any _0x76BBA2CEE66D47E9(Any p0);
+int _0x76BBA2CEE66D47E9(Ped ped);
 ```
 
```

Comment:
Mind putting some information about what the native does in the description as well? That way people can find it using the search feature (because there's no name for this native).